User Guide.md

```diff
@@ -1,29 +1,36 @@
+[View in GitHub ![GitHub Logo](images/GitHubLogo.png#right)](https://github.com/pln-planning-tools/StarMaps)
```

Comment:
We should add this link client side instead of updating the markdown directly

Reply:
moved out of md into index.tsx